contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L2DepositedERC20.sol

```diff
@@ -21,139 +20,38 @@ import { OVM_CrossDomainEnabled } from "../../../libraries/bridge/OVM_CrossDomai
  * Compiler used: optimistic-solc
  * Runtime target: OVM
  */
-contract OVM_L2DepositedERC20 is iOVM_L2DepositedERC20, UniswapV2ERC20, OVM_CrossDomainEnabled {
-    
-    /*******************
-     * Contract Events *
-     *******************/
+contract OVM_L2DepositedERC20 is Abs_L2DepositedERC20, UniswapV2ERC20 {
 
-    event Initialized(iOVM_L1ERC20Gateway _l1ERC20Gateway);
+    uint32 public constant MY_FINALIZE_WITHDRAWAL_L1_GAS = 100000;
```

Comment:
`Abs_L2DepositedERC20` also has `DEFAULT_FINALIZE_WITHDRAWAL_L1_GAS`, and `_getL1FinalizeWithdrawalGasCost()`, though there is a `todo` comment to remove the constant override.